ember-toucan-core/src/-private/components/control.hbs

```diff
@@ -1,3 +1,3 @@
-<div class="mt-1" ...attributes>
+<div ...attributes>
```

Comment:
Since we don't know what type of control or where the control is going to go, I thought it made more sense to _not_ add this and instead put it in the `*-field` components instead. Reason being, we don't want `mt-1` here for Checkbox, for example as that'll add unwanted top-margin.  Instead of controlling setting the margin with a component argument, I think it's okay to control these opinionated styles in each component, but I'm curious what y'all think. I also added the design guidance line above in the docs so it's documented what we are looking for.